independent-projects/bootstrap/runner/src/main/java/io/quarkus/bootstrap/runner/AugmentEntryPoint.java

```diff
@@ -0,0 +1,49 @@
+package io.quarkus.bootstrap.runner;
+
+import java.io.ObjectInputStream;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.net.URLDecoder;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Experimental class that allows Quarkus applications to be re-augmented
+ * 
+ */
+public class AugmentEntryPoint {
+
+    public static void main(String... args) throws Exception {
```

Comment:
So there is a mutability main entry and a reaugment main entry? What’s the difference?

Reply:
This is the entry point to regenerate the app. As my transparent rebuild idea was not popular this must be invoked manually. For now while this is experimental this is just run using java -cp quarkus-run.jar io.quarkus.bootstrap.runner.AumentEntryPoint, once this feature is no longer experimental we will likely want an easier way to do this.

Reply:
ok but what about this one? `MutabilityEntryPoint`

Reply:
Its just badly named, its purpose changed but not its name, I have renamed it.